javascript-functions-practice/get-average-of-three-numbers.js

```diff
@@ -0,0 +1,116 @@
+/* exported getAverageOfThreeNumbers */
+function getAverageOfTwoNumbers(x, y) {
```

Comment:
Make sure that you only have one function in each file, and that each function matches the file name